server/src/main/resources/db/changelog/20170220102027-add-id-column-for-productcontent.xml

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+        http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-2.0.xsd">
+
+    <changeSet id="20170220102027-1" dbms="mysql" author="crog">
+        <!-- On MySQL/MariaDB, we need to drop *all* foreign keys before we can drop our primary key -->
+        <dropAllForeignKeyConstraints baseTableName="cp2_product_content"/>
```

Comment:
I would like to see all of these SQL commands in one single `changeSet`.  I say that because it feels like all these statements should be within a single transaction.  The Liquibase docs say:

```
Liquibase attempts to execute each changeSet in a transaction that is committed at the end, or rolled back if there is an error. Some databases will auto-commit statements which interferes with this transaction setup and could lead to an unexpected database state. Therefore, it is usually best to have just one change per changeSet unless there is a group of non-auto-committing changes that you want applied as a transaction such as inserting data.
```

And I think in this particular case, there's a case to be made that we want these changes applied as a transaction so we don't end up with no constraints on a table or something